src/job.rb

```diff
@@ -6,8 +6,11 @@
 
 # Information representing a CI job.
 class Job
+  # rubocop:disable Layout/LineLength
+  # Splitting the long line up is probably just going to make things worse.
   NAME_REGEX =
-    /\A(?<runner>\d+(?:\.\d+)?(?:-arm64)?(?:-cross)?)-(?<run_id>\d+)(?:-(?<run_attempt>\d+))?(?:-(?<tags>[-a-z]+))?\z/
+    /\A(?<runner>\d+(?:\.\d+)?(?:-(?:arm|x86_)64)?(?:-cross)?)-(?<run_id>\d+)(?:-(?<run_attempt>\d+))?(?:-(?<tags>[-a-z]+))?\z/
```

Comment:
Thoughts on something like this?

```suggestion
    NAME_REGEX = %r{
      \A
        (?<runner>\d+(?:\.\d+)?(?:-(?:arm|x86_)64)?(?:-cross)?)
        -(?<run_id>\d+)
        (?:-(?<run_attempt>\d+))?
        (?:-(?<tags>[-a-z]+))?
      \z
    }x
```

Reply:
Forgot about `x`. Yea, that's a lot better.